src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/automatic-anomalies.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Automatic anomaly detection 
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how automatic automatic anomaly detection in New Relic notifies you of unusual app behavior. 
+---
+
+Automatic anomaly detection is the easiest way for you and your team to detect and analyze unusual behavior in your system. Automatic anomaly detection uses the following methods to detect anomalies in your app data:
+
+1. Anomaly detection monitors metric data reported by an APM agent, building a model of your typical application dynamics, and focuses on key golden signals: throughput, response time, and errors.
+2. If one of these golden signals shows anomalous behavior, the system flags the behavior and tracks recovery to normal behavior.
+3. The system adapts to changes in your data, and continuously updates models based on new data.
+
+**Automatically on**: By default, anomaly detection monitors all your APM applications, with no action required by you. When an anomaly is detected, it's automatically surfaced in various activity streams, the applied intelligence anomalies feed and is available for querying via NRQL.
+
+**Correlation**: Anomalies are connected to our correlation logic, otherwise known as decisions. If our correlation logic determines that issues are connected, then a single issue will be created from multiple incidents. You can see this correlated issue in the issues feed. Anomalies provide additional context about what changed around the time an issue occurred so your team can easily detect patterns and diagnose problems. 
+
+**Receiving notifications:** When New Relic detects anomalous changes in throughput, error rate, or response time, your team has the option to receive notifications about this behavior. We recommend sending notifications selected Slack channels or email channels. When the anomaly goes back to normal then a recovery message is sent. If you don't want to receive notifications you can still access to the data using a [NRQL query](#anomaly-events).
+
+**Anomaly analysis:** For each anomaly, we provide a link in Slack to an analyze anomaly page. This page generates automatic insights into the anomaly. The page is also available from the anomalies tab, which lists recent anomalies. This page uses your existing APM and anomaly detection data to provide explanations as to the cause of the anomaly.
+
+**Activity stream:** Inside various activity streams such as the New Relic homepage, APM Summary page, Lookout and Explorer, you'll see relevant anomalies from your APM-monitored applications. Clicking on any of the anomaly events in the activity stream brings up the analysis page for that anomaly.
+
+Applications will not always generate anomalies, so it can be normal to not receive any detections.
```

Comment:
"Won't" or "don't" instead of will not, I've been leaning away from "will" statements so I'd lean towards "don't", but it's 100% your call.

Reply:
done